modules/attack.ts

```diff
@@ -36,6 +36,11 @@ export async function runAttacksOnLocalTarget(team: string): Promise<[string, st
                     alerts.push(message);
                 }
 
+                const vuln = lineBuf.match(/POTENTIAL VULNERABILITY: (.+)/)?.[0];
```

Comment:
Should be `?.[1]` in this case :) the match array returned by `.match` is of the form
```
[entire match, ...groups]
```
and I assume you want to extract just the group here.